utils.js

```diff
@@ -0,0 +1,38 @@
+const DatArchive = require('node-dat-archive')
+
+exports.checkUserArchiveForApprovalOfPub = async function(
+  applicationPub,
+  applicationKey,
+  userUrl
+) {
+  const userArchive = new DatArchive(userUrl, {
+    datOptions: { latest: true }
+  })
+
+  const proofLocation = `/.well-known/dat-pubs/${applicationPub.url.slice(
+    6
+  )}.json`
+
+  let file
+  try {
+    file = await userArchive.readFile(proofLocation)
+  } catch (err) {
+    throw new Error(
+      `Proof file missing: /.well-known/dat-pubs/${applicationPub.url.slice(
```

Comment:
Should replace this with `Proof file missing: ${proofLocation}` to make it easier to change the location in the future.

Reply:
Done